bluesky/traffic/route.py

```diff
@@ -69,7 +85,7 @@ def addwptStack(self, idx, *args):  # args: all arguments of addwpt
                 return True
 
         # Convert to positions
-        name = args[0]
+        name = args[0].upper().strip()

```

Comment:
The stripping/uppering is done several times. May as well do it once at the start.